tasks/initial_clean.yml

```diff
@@ -22,16 +25,29 @@
       src: "{{ item.src }}"
       dest: "{{ item.dest }}"
     with_items:
-      - { src: /usr/share/abrt/conf.d/abrt-action-save-package-data.conf, dest: /etc/abrt/abrt-action-save-package-data.conf }
-      - { src: /usr/share/abrt/conf.d/abrt.conf, dest: /etc/abrt/abrt.conf }
-      - { src: /usr/share/abrt/conf.d/plugins/CCpp.conf, dest: /etc/abrt/plugins/CCpp.conf }
-      - { src: /usr/share/abrt/conf.d/plugins/vmcore.conf, dest: /etc/abrt/plugins/vmcore.conf }
+      - {
+          src: /usr/share/abrt/conf.d/abrt-action-save-package-data.conf,
+          dest: /etc/abrt/abrt-action-save-package-data.conf
+        }
+      - {
+          src: /usr/share/abrt/conf.d/abrt.conf,
+          dest: /etc/abrt/abrt.conf
+        }
+      - {
+          src: /usr/share/abrt/conf.d/plugins/CCpp.conf,
+          dest: /etc/abrt/plugins/CCpp.conf
+        }
+      - {
+          src: /usr/share/abrt/conf.d/plugins/vmcore.conf,
+          dest: /etc/abrt/plugins/vmcore.conf
+        }
   - name: Restart abrtd service
     service:
       name: abrtd
       state: restarted
   - name: Drop libvirt sasl2 configuration by vdsm
-    command: sed -i '/## start vdsm-4.[0-9]\+.[0-9]\+ configuration/,/## end vdsm configuration/d' /etc/sasl2/libvirt.conf
+    command:
```

Comment:
here too. sorry I missed it.

Reply:
In theory '>-' is neede only if we break over multiple lines but let's try to keep it more consistent